src/cache/event.rs

```diff
@@ -597,11 +598,12 @@ impl CacheUpdate for ReadyEvent {
         let mut guilds_to_remove = vec![];
         let ready_guilds_hashset =
             self.ready.guilds.iter().map(|status| status.id).collect::<HashSet<_>>();
-        let shard_data = self.ready.shard.unwrap_or([1, 1]);
+        let shard_data = self.ready.shard.unwrap_or_else(|| ShardInfo::new(1, 1));
```

Comment:
The ids start with `0`, right?
```suggestion
        let shard_data = self.ready.shard.unwrap_or_else(|| ShardInfo::new(0, 1));
```

Reply:
I didn't want to change the existing behavior of the code, just refactor how it currently works, but if this is a bug I can get rid of new, impl default on shardinfo with `0, 1` and use that

Reply:
You're right. It's better to keep it as it is.